.lintstagedrc

```diff
@@ -1,12 +1,10 @@
 {
-  "*.css": ["prettier --write", "stylelint --fix", "git add"],
-  "*.scss": ["prettier --write", "stylelint --syntax=scss --fix", "git add"],
+  "*.css": ["prettier --write", "stylelint --fix"],
```

Comment:
Why removing the `git add` command after lint fixed the issues?

Reply:
I think it wasn't needed anymore as lint-staged already does for you. Anyway this package is not used anymore so no need to worry